### DISCUSSION TITLE
Load More

### DISCUSSION THREAD

**Discussion by silicahd** [livewire/livewire#1551] [4. General]
Pagination is nice but i think a load more function would a really nice touch. I guess we can use alpine to achieve that but still would be nice to have two options to pick from.

**Comment by ju5t** [comment]
You don't need Alpine for this. You just create and call a custom method like `loadMore()` on your Livewire component. That would update the query and increases the current limit of your query by X amount.

It is something you need to make yourself, but it's pretty easy.

**Reply by silicahd** [reply]
Thank you for the reply that is a step in the right direction, I honestly just found livewire and alpine and have been at it for the past week trying to learn it. So thank you.
